build_and_push_nexus.sh

```diff
@@ -19,10 +19,10 @@ checkTag $TAG "${tags[@]}"
 rm build/libs/notary-1.0-SNAPSHOT-all.jar || true
 gradle shadowJar
 
-docker build -t nexus.iroha.tech:19002/d3-deploy/eth-relay:$TAG -f eth-relay.dockerfile .
-docker build -t nexus.iroha.tech:19002/d3-deploy/registration:$TAG  -f registration.dockerfile .
-docker build -t nexus.iroha.tech:19002/d3-deploy/notary:$TAG  -f notary.dockerfile .
-docker build -t nexus.iroha.tech:19002/d3-deploy/withdrawal:$TAG  -f withdrawal.dockerfile .
+docker build -t nexus.iroha.tech:19002/d3-deploy/eth-relay:$TAG -f docker/eth-relay.dockerfile .
+docker build -t nexus.iroha.tech:19002/d3-deploy/registration:$TAG  -f docker/registration.dockerfile .
+docker build -t nexus.iroha.tech:19002/d3-deploy/notary:$TAG  -f docker/notary.dockerfile .
+docker build -t nexus.iroha.tech:19002/d3-deploy/withdrawal:$TAG  -f docker/withdrawal.dockerfile .
```

Comment:
don't use scripts to build docker images. there is an easier way:
https://github.com/soramitsu/BCA-Sora-Unit-Backend/blob/develop/build.gradle#L14
https://github.com/soramitsu/BCA-Sora-Unit-Backend/blob/develop/build.gradle#L149-L155
https://github.com/soramitsu/BCA-Sora-Unit-Backend/blob/develop/Dockerfile
https://github.com/soramitsu/BCA-Sora-Unit-Backend/blob/develop/Jenkinsfile#L21